types/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "esnext",
+    "lib": ["es6"],
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
+    "noEmit": true,
+    "jsx": "react",
```

Comment:
Nit: technically this option is unnecessary since the library does not use JSX

Reply:
It's needed for test.tsx since it contains JSX

Reply:
Good point, thanks for letting me know!